engine/src/main/java/org/destinationsol/CommonDrawer.java

```diff
@@ -147,11 +153,11 @@ private void setTint(Color tint) {
         spriteBatch.setColor(tint);
     }
 
-    public void draw(TextureRegion tex, Rectangle rect, Color tint) {
-        draw(tex, rect.width, rect.height, (float) 0, (float) 0, rect.x, rect.y, (float) 0, tint);
+    public void draw(TextureRegion tex, Rectangle rect, Color tint,boolean addative) {
```

Comment:
Typo, or is "addative" used on purpose here and in a few other places?

Also minor style issue with whitespace here and elsewhere